.gitignore

```diff
@@ -18,6 +18,7 @@ spec/reports/pacts
 .yarn/cache/
 .yarn/install-state.gz
 .nyc_output
+.byebug_history
```

Comment:
I think that as this change is not related to Get Involved pages, it should either be in it's own commit, or even better, added in a separate PR.